divu.asm

```diff
@@ -0,0 +1,5 @@
+and $t1 $t1 $0
+and $t2 $t2 $0
+ori $t2 $t2 255
+ori $t1 $t1 5
+divu $t2 $t1
```

Comment:
you may comment the result after every line and its machine code as in [here](https://github.com/abdo1819/arch_lab09/blob/master/mipstest.asm)